.gitignore

```diff
@@ -45,3 +45,4 @@ output/*/index.html
 
 # Sphinx
 docs/_build
+.idea/
```

Comment:
Best practice so that you don't have to do this on every project you ever work on is to set up a global gitignore: https://stackoverflow.com/a/7335487/1427135